src/panels/lovelace/components/hui-conditional-base.ts

```diff
@@ -67,6 +77,10 @@ export class HuiConditionalBase extends ReactiveElement {
         this.appendChild(this._element);
       }
     }
+
+    if (visible !== oldVisibility) {
+      fireEvent(this, "update-layout");
```

Comment:
Should events be named for the firer instead of the subscriber? i.e. "visibility-changed" instead of "update-layout"?